app.py

```diff
@@ -3,33 +3,29 @@
 import os
 import json
 from pathlib import Path
-from flask import Flask, render_template
+from flask import Flask
+
+# Importing Modules
+import helper_functions as f
 
 app = Flask(__name__) #creating Flask class object
 title='Quotation'
 
 @app.route('/') #decorator defines the
 @app.route('/home')
-@app.route('/base')
 @app.route('/index')
```

Comment:
Too many URLs for a single functions. Earlier we were using HTML templates but now we can remove those. Only keep the base URL.

Reply:
Keeping only 2 URLs ("/") and ("/home")